src/mail.rs

```diff
@@ -228,38 +229,55 @@ pub async fn send_single_org_removed_from_org(address: &str, org_name: &str) ->
     send_email(address, &subject, body_html, body_text).await
 }
 
+// The url cannot be fully built using `url.Url` since the anchor `#` should be after the query parameters
```

Comment:
What does this comment mean exactly? Aren't you replacing the `/?email` with `/#/accept-organization/?email below? (Is this related to my other question? If so I would move the comment down before the replace.)

Reply:
It is, by definition the anchor `#` should be after the query parameters.
So if set at the start you'll end up with an url: `domain.com/?token=toto#/accept-organization/`

Reply:
This comment should be moved down then (and maybe also reworded a bit as it's saying the opposite to me).
```suggestion
// The url cannot be fully built using `url.Url` since the anchor `#` would be after the query parameters so we use `replace` as a work around here
```

Reply:
Do you prefer it just before the `Url::parse` or lower down before the `replace` ?

Reply:
If possible I would add the comment directly above the replace (as this is where I was wondering why it's done this way and where I think it would be most relevant).

Reply:
Realized that the `replace` was invalid if the domain included a subpath.
Could have just removed the `/  but I think this new version should be safer.